package.json

```diff
@@ -21,7 +21,8 @@
     }
   ],
   "dependencies": {
-    "event-kit": "^1.0.3"
+    "event-kit": "^1.0.3",
+    "underscore-plus": "^1.6.6"
```

Comment:
I'd prefer not to add underscore-plus as a dependency. Maybe we could use an ES6 [String::repeat polyfill](https://github.com/mathiasbynens/String.prototype.repeat) instead.